src/ncp/ncp_base.cpp

```diff
@@ -4716,6 +4729,158 @@ ThreadError NcpBase::SetPropertyHandler_MAC_WHITELIST_ENABLED(uint8_t header, sp
     return errorCode;
 }
 
+#if OPENTHREAD_ENABLE_RAW_LINK_API
+
+ThreadError NcpBase::SetPropertyHandler_MAC_SRC_MATCH_ENABLED(uint8_t header, spinel_prop_key_t key,
+                                                              const uint8_t *value_ptr, uint16_t value_len)
+{
+    bool isEnabled;
+    spinel_ssize_t parsedLength;
+    ThreadError errorCode = kThreadError_None;
+
+    parsedLength = spinel_datatype_unpack(
+                       value_ptr,
+                       value_len,
+                       SPINEL_DATATYPE_BOOL_S,
+                       &isEnabled
+                   );
+
+    if (parsedLength > 0)
+    {
+        errorCode = otLinkRawSrcMatchEnable(mInstance, isEnabled);
+        if (errorCode == kThreadError_None)
+        {
+            errorCode = HandleCommandPropertyGet(header, key);
+        }
+        else
+        {
+            errorCode = SendLastStatus(header, ThreadErrorToSpinelStatus(errorCode));
+        }
+    }
+    else
+    {
+        errorCode = SendLastStatus(header, SPINEL_STATUS_PARSE_ERROR);
+    }
+
+    return errorCode;
+}
+
+ThreadError NcpBase::SetPropertyHandler_MAC_SRC_MATCH_SHORT_ADDRESSES(uint8_t header, spinel_prop_key_t key,
+                                                                      const uint8_t *value_ptr, uint16_t value_len)
+{
+    ThreadError errorCode = kThreadError_None;
+    spinel_status_t errorStatus = SPINEL_STATUS_OK;
+    const uint8_t *data = value_ptr;
+    uint16_t data_len = value_len;
+
+    // Clear the list first
+    errorCode = otLinkRawSrcMatchClearShortEntries(mInstance);
+
+    VerifyOrExit(errorCode == kThreadError_None,
+        errorStatus = ThreadErrorToSpinelStatus(errorCode));
+
+    // Loop through the addresses and add them
+    while (data_len >= sizeof(uint16_t))
+    {
+        spinel_ssize_t parsedLength;
+        uint16_t short_address;
+
+        parsedLength = spinel_datatype_unpack(
+            data,
+            data_len,
+            "S",
```

Comment:
Using define `SPINEL_DATATYPE_UINT16_S` instead of `"S`" would be better here.
I know it's not consistent in the existing code, but I'd say for any new code let's use the defines (makes the code cleaner and more readable).